config/environments/dev_privileged_libvirt.yml

```diff
@@ -2,9 +2,9 @@
 # you can use these options.  This also makes VMs survive reboots \o/
 # This is particularly useful for local development environments
 non_root_chown: true
-non_root_user: stack
-undercloud_user: stack
-working_dir: "/home/{{ non_root_user }}"
+non_root_user: $USER
```

Comment:
This won't work, will it?  Doesn't this need to be a j2 reference to an ansible variable?

E.g something like {{ansible_user_id}} ?

Reply:
I think this is a refuse from a wrong commit, I might have messed up something with my local repo.
I actually used {{ lookup('env','USER') }} since I think {{ansible_user_id}} will give root

Reply:
You could just pass -e non_root_user=$USER here:

https://github.com/metalkube/dev-scripts/blob/master/02_configure_host.sh#L12

Reply:
So remove all non_root_user definitions from the playbooks ?
We also have undercloud_user and ssh_user using the same variable, maybe reference the value of non_root_user instead ?

Reply:
No I'm saying leave non_root_user alone and just pass the value you want to ansible-playbook :)